lib/utils.js

```diff
@@ -117,9 +117,11 @@ exports.scanUrls = function(urls, opts) {
       // Send REST request.
       return new Promise(function(fulfill, reject) {
 
-        // If URL contains a wildcard then immediately set fufill with yellow status
+        // If URL contains a wildcard then immediately set fulfill with yellow
+        // status
         if (_.includes(url, '*')) {
           fulfill({url: url, status: true, color: 'yellow'});
+          return;
```

Comment:
wondering if it's preferable (and if there is a difference) between this and just `return  fulfill({url: url, status: true, color: 'yellow'});`